### PR TITLE
inscription et connection

### PR DESCRIPTION
**Pull request by Gumatan**
backend auth routes, frontend backend link in signup component, added redux to store JWT

### CODE REVIEW

**Review by ghost — DISMISSED** [review]
RAS

File: backend/conf.js [hunk 1]
```diff
@@ -10,6 +17,6 @@ const db = mysql.createPool({
 });
 
 module.exports = {
-  backendPort,
+  CONFIG,
```
Comment (LoicBrassart) [inline backend/conf.js:29]: (Hmm, I don't really like the CONFIG sub-object with one part of the variables, but it works and doesn't break anything)

File: frontend/src/components/Form.jsx [hunk 2]
```diff
@@ -37,80 +37,17 @@ function Form() {
       transitionLeave={true}
     >
       <container className="formContainer">
-        <form className="form" onSubmit={e => handleSubmit(e)}>
-          {/* LastName -------------------------------------------------------------------------------------------- */}
+        <form
+          className="form"
+          onSubmit={e => {
```
Comment (LoicBrassart) [inline frontend/src/components/Form.jsx:42]: (`handleSubmit` should be called from here, it'd prevent you from firing another event on the validation button click. Once again, it's not blocking and only a minor performance matter, I'll let it pass for now)

**Review by LoicBrassart — DISMISSED** [review]
Minor evils, I'll let it pass

**Review by ghost — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**